src/styles/abstract/_font.scss

```
@@ -0,0 +1,3 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&display=swap');
+
+$primary-font: 'Open Sans', sans-serif;
```

```suggestion
body{
  font-family: 'Open Sans', sans-serif;
  font-optical-sizing: auto;
  font-weight: 400;
  font-style: normal;
  font-variation-settings: 'wdth' 100;
}
```

then import this file into main.scss then we will have a global font styling